README.md

```diff
@@ -1,80 +1,74 @@
+[![Twitter](https://img.shields.io/badge/-Twitter-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://twitter.com/learnanything_)
+[![Spectrum](https://img.shields.io/badge/-Spectrum-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://spectrum.chat/learn-anything)
+[![Paper](https://img.shields.io/badge/-Paper-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://github.com/learn-anything/learn-anything/wiki/White-Paper)
+[![Support](https://img.shields.io/badge/%E2%9D%A4-Support-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://www.patreon.com/learnanything)
 
-[Learn Anything](https://learn-anything.xyz/) is an open source website to organize world's knowledge with study plans for learning anything.
+#### [Learn Anything](https://learn-anything.xyz) is a project to organize world's knowledge in a mind map and curate study plans for learning anything.
 
-The goal of the website is to accelerate the rate at which people learn, by giving the most efficient paths one can take to gain full understanding of any given topic.
+### Curated Lists
 
-For more information, you can read our [White Paper](https://github.com/learn-anything/learn-anything/wiki/White-Paper), check [our wiki](https://github.com/learn-anything/learn-anything/wiki), ask questions on [Spectrum](https://spectrum.chat/learn-anything) and join our [Discord server](https://discord.gg/KKYdWjt).
+With our goal of providing the most efficient paths for Learning Anything, we also manage curation of various curated lists. Feel free to [explore and improve them](https://github.com/learn-anything/curated-lists#readme).
 
-## Contributing
-
-Since this website is fully open source, you can help and add new and awesome features to the website. Please read our [contribution guidelines](CONTRIBUTING.md#readme) for different ways on how you can do this.
+### API
 
-If you have never programmed before and this is your first open source project, don't be afraid to join in. There are many people that can help you both get started and make your first contribution.
+[Here](http://docs.learn-anything.xyz) you can find the public API endpoints we currently have. We will be adding more in the future.
 
-## New design
+Below are some things people have built with it. Feel free to add yours to this list.
 
-Here are some mockups kindly made by [Han Lyn](https://github.com/iinfin) for the next version of Learn Anything.
+- [Alfred Workflow](https://github.com/nikitavoloboev/alfred-learn-anything) - Alfred workflow to search through all Learn Anything maps.
+- [Albert Plugin](https://github.com/nglgzz/albert-plugins) - Albert plugin to search through all Learn Anything maps.
 
-![](https://cdn.discordapp.com/attachments/493483373478543360/493486353795907626/01-home.png)
+## Contributing
 
-> Home page
+Please read our [contribution guidelines](CONTRIBUTING.md#readme).
 
-![](https://cdn.discordapp.com/attachments/493483373478543360/493486372233936907/02-about.png)
+## New Design
```

Comment:
I would move New Design above too as with merging of this PR, we can make 'latest' be our default branch. If @nglgzz agrees too.

So it would be Title (description) -> New Design (draw attention to designs and get people to help out) -> Contributing -> Curated Lists -> API -> Support -> Thank you -> Team

Reply:
I would wait making it default until we have at least some boilerplate code on latest.